### PR TITLE
Upgraded django version

### LINKED ISSUE #12: upgrade django version
django.middleware.common.CommonMiddleware in Django 1.11.x before 1.11.15 and 2.0.x before 2.0.8 has an Open Redirect.
Thus we need to upgrade it to 2.0.8 or later

### PR DESCRIPTION
**Pull request by Sai-Adarsh**
upgraded django version requirements.txt and pipfile
Fixes #12

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ url = "https://pypi.python.org/simple"
 verify_ssl = true
 
 [packages]
-django = "*"
+django = "2.1.3"
```
Comment (Monal5031) [inline Pipfile]: No don't change this. Not needed

**Review by Monal5031 — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (Sai-Adarsh) [reply Pipfile]: okay

Comment (Sai-Adarsh) [reply Pipfile]: > No don't change this. Not needed

Done. check the PR

**Review by Monal5031 — APPROVED** [review]
LGTM :+1: great work with the digging that pip lock is already updated :tada:

**Comment by Monal5031** [comment]
@Sai-Adarsh make sure to have your commits squashed next time :)

**Pull request merged**